### DISCUSSION TITLE
Water consumption management

### DISCUSSION THREAD

**Discussion by dougiteixeira** [home-assistant/frontend#11058] [Other feature requests]
Include water consumption management similar to energy and gas management.

**Comment by MindFreeze** [comment]
We have this now. Closing

**Reply by dxmnkd316** [reply]
Where can we find this?  Unless I missed a new feature rollout or a PR, I only see the PR you had submitted but was closed without merging.

**Reply by MindFreeze** [reply]
This request is for general water source tracking which we have, not for a hierarchy like energy

**Reply by dxmnkd316** [reply]
But the request was literally "Include water consumption management **_similar to energy and gas management_.**"  We don't have that unless I am missing something.  Not trying to be a pain, just trying to understand here.  Thanks

**Reply by MindFreeze** [reply]
It is similar to energy and gas management. If you want "individual water devices", there are other discussions on this and I am actually working on it, but it is out of scope here
